### PR TITLE
LPS-142445 Should not display entries created by other user

### PR DESCRIPTION
**Pull request by MarinhoFeliphe**

### CODE REVIEW

**Comment by MarinhoFeliphe** [comment]
ci:test:relevant

**Comment by rafaprax** [comment]
@MarinhoFeliphe please add a test to cover this scenario. Thx

File: modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowInstanceManagerImplTest.java [hunk 1]
```diff
@@ -158,6 +160,51 @@ public void testSearchWorkflowInstancesWhenThereIsAnUnregisteredHandler()
...
+		throws Exception {
+
+		WorkflowDefinition workflowDefinition =
+			_workflowDefinitionManager.deployWorkflowDefinition(
+				TestPropsValues.getCompanyId(), TestPropsValues.getUserId(),
+				RandomTestUtil.randomString(), RandomTestUtil.randomString(),
+				FileUtil.getBytes(
+					getResourceInputStream(
+						"single-approver-workflow-definition.xml")));
+
```
Comment (rafaprax) [inline modules/apps/portal-workflow/portal-workflow-kaleo-test/src/testIntegration/java/com/liferay/portal/workflow/kaleo/internal/runtime/integration/test/WorkflowInstanceManagerImplTest.java:174]: Hey @MarinhoFeliphe, you don't need to deploy a new single approver definition. It is enough to use the existent one, like you did in the `testSearchWorkflowInstancesWhenThereIsAnUnregisteredHandler`.

Thx

**Review by rafaprax — CHANGES_REQUESTED** [review]

**Comment by MarinhoFeliphe** [comment]
ci:forward

**Comment by brianchandotcom** [comment]
@MarinhoFeliphe I merged this manually with your latest changes from 7 minutes ago. Thx!

**Comment by rafaprax** [comment]
manually merged

**Pull request closed without merging**